src/index.jsx

```diff
@@ -1,15 +1,19 @@
 import React from 'react';
 import ReactDOM from 'react-dom';
-import { BrowserRouter, Route } from 'react-router-dom';
+import { Route } from 'react-router-dom';
+import { BrowserRouter } from 'react-g-analytics';
 import Site from './components/Site/Site';
 
-// TODO: Re-integrate <GoogleAnalytics analyticsId="UA-46921629-2" />
-// Consider `react-g-analytics` package
+let gaID;
+
+if(process.env.NODE_ENV === 'production') {
+  gaID = 'UA-46921629-2';
+}
 
 // Client Side Rendering
 if ( window.document !== undefined ) {
   ReactDOM.render((
-    <BrowserRouter>
+    <BrowserRouter id={gaID}>
```

Comment:
This component triggers an error locally because `id` is required. I think it is fine, haven't tried passing an empty string, or any string for that matter.

Reply:
Yeah, maybe default `gaID` to an empty string and see what happens? If not, we should change it to something like:

``` js
import { BrowserRouter, Route } from 'react-router-dom'
import { BrowserRouter as AnalyticsRouter } from 'react-g-analytics'

if ( process.env.NODE_ENV === 'production' ) {
  BrowserRouter = AnalyticsRouter
}
```

Then in dev, the normal `BrowserRouter` will simply ignore the `id` prop.

Reply:
That would be cool, but `import`s are defined as `const`

Reply:
What about:
```
const BrowserRouter = process.env.NODE_ENV === 'production' ? require('react-g-analytics') : require('react-router-dom').BrowserRouter;
```

Reply:
yes or if we want imports just define another variable:
```
Router = process.env.NODE_ENV === 'production' ? AnalyticsRouter : BrowserRouter;
...
<Router ...>
```